src/contentscript/overlay/root/components/OverlayTab/index.tsx

```diff
@@ -141,6 +160,14 @@ export const OverlayTab = (p: OverlayTabProps): ReactElement => {
     const url = `${DAPPLETS_STORE_URL}/#searchQuery=${f}`
     window.open(url, '_blank')
   }
+  const getNotifications = async () => {
```

Comment:
Line break

Reply:
I use linters' scripts and lines deleted